src/commands/synthetics/run-test.ts

```diff
@@ -155,8 +157,11 @@ export const executeTests = async (
       tunnel
     )
 
+    const {orgMaxConcurrencyCap} = await getSettings(api)
```

Comment:
Why fetch it in `executeTests` if we're not using it? The function is already quite complex, we could move that to `execute` directly IMO and get rid of the mutable as well